README.md

```diff
@@ -139,8 +139,11 @@ You can configure this template by passing different parameters in the Generator
   - use "-p language=java" option in the client code generation command line.
   - once the client code is generated <br>
 &nbsp;&nbsp;&nbsp;&nbsp; cd output <br>
+&nbsp;&nbsp;&nbsp;&nbsp; mkdir -p src/main/java/asyncapi/websocket/client/template <br>
+&nbsp;&nbsp;&nbsp;&nbsp; mv AsyncapiWebSocketClientEndpoint.java src/main/java/asyncapi/websocket/client/template <br>
 &nbsp;&nbsp;&nbsp;&nbsp; mvn package<br>
```

Comment:
assume that if we modify the java file name, would it be possible to get rid of the two steps (mkdir and mv)?